backend/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/cilium/hubble-ui/backend
 go 1.14
 
 require (
+	github.com/cenkalti/backoff/v4 v4.1.0
```

Comment:
We have exponential backoff in cilium. Not sure if you looked at it, but just in case if you haven't seen it.

Reply:
didnt know, thx
then I will rewrite this code using cilium's retries

Reply:
https://godoc.org/github.com/cilium/cilium/pkg/backoff
this package?
looks very minimalistic + it seems like there is no `Retry` method for using that

Reply:
@yandzee yes that's the package.

retry is basically calling `Wait` https://godoc.org/github.com/cilium/cilium/pkg/backoff#Exponential.Wait and then calling this function: https://github.com/cilium/hubble-ui/pull/117/files#diff-2395afde28d649c46a74a24d548789a3384fbca13e5ef45e71040ac7f8e0c7cdR115

Reply:
You don't necessarily have to is it if you like the ergonomics of the current package better or there is some missing functionality. I was just suggesting a bit of consolidation of deps if it makes sense